src/devtools/client/webconsole/components/FilterBar/FilterBar.js

```diff
@@ -355,4 +331,5 @@ module.exports = connect(mapStateToProps, {
   filterBarDisplayModeSet: actions.filterBarDisplayModeSet,
```

Comment:
We can get rid of groupWarnings in `mapStateToProps`

Reply:
yep